.github/scripts/interact-with-contract.sh

```diff
@@ -14,7 +14,7 @@ $CC cargo contract new --target-dir /sources flipper
 $CC cargo contract build --release --manifest-path=/sources/flipper/Cargo.toml
 
 # Upload and instantiate code
-ADDRESS=$($CC cargo contract instantiate --suri //Alice --skip-confirm --args false -x --output-json /sources/flipper/target/ink/flipper.wasm | jq -jr '.contract')
+ADDRESS=$($CC cargo contract instantiate --suri //Alice --url ws://localhost:9933 --skip-confirm --args false -x --output-json /sources/flipper/target/ink/flipper.wasm | jq -jr '.contract')
```

Comment:
should be 9944 I guess, since that is what you're exposing?

Reply:
```suggestion
ADDRESS=$($CC cargo contract instantiate --suri //Alice --url ws://localhost:9944 --skip-confirm --args false -x --output-json /sources/flipper/target/ink/flipper.wasm | jq -jr '.contract')
```